eng/versioning/version_data_java_files.txt

```diff
@@ -0,0 +1,8 @@
+# This is necessary because there are java files with a hard coded version in them that need to
```

Comment:
Would it make things easier to just have one file list? So folks don't have to try and figure out which one to put the file in?

Reply:
@weshaggard Right now the versions strings in client files are client versions and the version strings in data files are data versions. I went the same way I'd done with the initial version text files. If there's confusion about what goes where a person could look at which version file their artifact is in and put their filename in the matching version_[client|data]_java_files.txt file.